trillian_log_api.proto

```diff
@@ -85,13 +85,16 @@ service TrillianLog {
     // Returns the total number of leaves that have been integrated into the
     // given tree. Corresponds to the ReadOnlyLogTreeTX.GetSequencedLeafCount
     // storage interface.
+    // DO NOT USE - FOR DEBUGGING/TEST ONLY (FIXME: remove or move to a debugging interface)
```

Comment:
Could we have the FIXME as a TODO please

Reply:
I'll remove the FIXME part for the moment as it's in use by the MySQL tests so we can't just delete it.